app/bundles/CoreBundle/Tests/Unit/Templating/Helper/DateHelperTest.php

```diff
@@ -110,4 +110,12 @@ function (string $key, array $parameters = []) {
 
         $this->assertSame('00:00:00', $this->helper->toText($dateTime));
     }
+
+    public function testFullConcat()
+    {
+        date_default_timezone_set('Europe/Paris');
```

Comment:
Setting time zone this way, will set it for other test cases as well.

Reply:
I suppose timezone is reset to old on each test by tearDownAfterClass

https://github.com/mautic/mautic/pull/9747/files/dec7d4456ef73d76d25482632bea72a8b57fbbd6#diff-94502daac727de55e303995909aa3441edad6e827d5e027bc1ee84ac9114910bR45

`date_default_timezone_set` is used in other 4 tests in that class.


Don't see issue.

Reply:
Sorry, I forgot to add the example test cases earlier which I had written to prove what I meant.

Here is the screenshot of an example:
![Screenshot 2021-04-19 at 13 57 07](https://user-images.githubusercontent.com/54406786/115207953-b225ad00-a119-11eb-9c96-b30ceae91fd2.png)

In test case 1 (testDefaultTimeZone), I checked for the default timezone set (which was UTC).
After this I set it to Europe/Paris.

In test case 2 (testSetTimeZoneInOtherTestCase), I again checked for the default timezone and it was Europe/Paris.

As you can see, both the test cases pass.

Reply:
Also, the updated time zone after it is set, won't affect test cases in another Test.php file though.
If this is okay, I do not see an issue.

Reply:
I understand, but If you check https://github.com/mautic/mautic/blob/features/app/bundles/CoreBundle/Tests/Unit/Templating/Helper/DateHelperTest.php

date_default_timezone_set is apply in each method


And https://github.com/mautic/mautic/blob/features/app/bundles/CoreBundle/Tests/Unit/Templating/Helper/DateHelperTest.php#L45

set timezone back to default after all classes method passes

Reply:
Yes, understood!
I see no issues. 👍🏻